engine/apps/mobile_app/tasks.py

```diff
@@ -208,7 +208,9 @@ def _get_fcm_message(alert_group, user, registration_id, critical):
                     alert=ApsAlert(title=alert_title, subtitle=alert_subtitle, body=alert_body),
                     sound=CriticalSound(
                         # The notification shouldn't be critical if the user has disabled "override DND" setting
-                        critical=(critical and mobile_app_user_settings.important_notification_override_dnd),
```

Comment:
just leaving a paper trail here in case others come across this. The in-code documentation for the `CriticalSound` class mentions that `critical` should be a boolean. However, the iOS [docs](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2990112) say `True/False`.

The commit prior to this, [ed5b5e153d289d355fbd529b39bf5cb66cafdebd](https://github.com/grafana/oncall/commit/ed5b5e153d289d355fbd529b39bf5cb66cafdebd), had this set to 0/1 and it appeared to be working

Reply:
Yeah, https://firebase.google.com/docs/reference/admin/python/firebase_admin.messaging#firebase_admin.messaging.CriticalSound mentions it should be boolean and there's how the lib handles the flag: https://github.com/firebase/firebase-admin-python/blob/master/firebase_admin/_messaging_encoder.py#L584-L585.
@imtoori do you mean that notifications are critical even when `important_notification_override_dnd` is set to `False`?